models/articles.model.js

```diff
@@ -66,3 +67,30 @@ exports.updateArticleById = (articleId, increaseVotesBy) => {
       return response.rows[0];
     });
 };
+
+exports.insertCommentByArticleId = (articleId, username, body) => {
+  if (
+    username === undefined ||
+    body === undefined ||
+    typeof body !== "string" ||
+    typeof username !== "string"
+  ) {
+    return Promise.reject({
+      status: 400,
+      message: "Bad request: request body invalid",
+    });
+  }
+  const createdAt = new Date();
+
+  return db
+    .query(
+      `
+      INSERT INTO comments (body, votes, author, article_id, created_at)
+      VALUES ($1, 0, $2, $3, $4)
+      RETURNING *;`,
+      [body, username, articleId, createdAt]
+    )
+    .then(({ rows }) => {
+      return rows[0];
```

Comment:
in seed.js, created_at has a default value NOW(), don't need to be insert in the model